tests/integration/api_admin_org_test.go

```diff
@@ -21,7 +21,7 @@ import (
 func TestAPIAdminOrgCreate(t *testing.T) {
 	onGiteaRun(t, func(*testing.T, *url.URL) {
 		session := loginUser(t, "user1")
-		token := getTokenForLoggedInUser(t, session)
+		token := getTokenForLoggedInUser(t, session, "sudo")
```

Comment:
Use the constant `AccessTokenScopeSudo` here and everywhere else in the tests.

Reply:
Ok. Since I'm using snake_case syntax in the web, e.g. `read_org` instead of `read:org` so that it would be automatically parsed based on `NewAccessTokenForm` struct in `services/forms/user_form.go`. Then it's not really the same as constants for some scopes, so I kept them in strings instead.
Does POST form request supports using `:` though? If it does, then it's possible to change everything.

Reply:
You are right but I have another idea:

Use `<input name="scope" class="enable-system" type="checkbox" value="repo:status">` and change the form to `Scope []string`. Then you don't need all these bool fields and `GetScope()` can be much simpler because you don't need reflection. You already have a string array and are able to filter the invalid values. Maybe you don't need join and split again the strings. And then you can use the constants in the tests.

Reply:
Implemented in 50a3c5f0d0cabd7298e73abf6cf2aa10bfae8b2e